README.md

```diff
@@ -105,6 +105,7 @@ Terminal Emulators
 - [PowerShell](https://github.com/PowerShell/PowerShell) - PowerShell for every system!
 - [xonsh](https://github.com/xonsh/xonsh) - 🐚 Python-powered, cross-platform, Unix-gazing shell
   [Z Shell](https://sourceforge.net/p/zsh/code/ci/master/tree/) - Zsh is a shell designed for interactive use, although it is also a powerful scripting language. [https://www.zsh.org/](https://www.zsh.org/)
+- [x-cmd](https://github.com/x-cmd/x-cmd) A vast and interesting collection of tools that can then bootstrap lots of other programs / functions in a consistent and structured way.
```

Comment:
Hey, thnx for the contribution. This should be in alphabetical order and in this case there should be a new Language category Awk with this tool included.. as it's written in awk

Awx
x-cmd

Reply:
Yes, I messed up the sorting, and this should be placed under tools. x-cmd is primarily written using posix shell, so it might be more appropriate to put it in the sh section.

I've moved the location of x-cmd in the README.

Reply:
great! thnx!